ivadomed/loader/adaptative.py

```diff
@@ -703,8 +703,8 @@ def HDF5_to_Bids(HDF5, subjects, path_dir):
     hdf5 = h5py.File(HDF5, "r")
     # check the dir exists:
     if not path.exists(path_dir):
-        print("Directory doesn't exist. Stopping process.")
-        exit(0)
+        raise FileNotFoundError("Directory doesn't exist. Stopping process.")
```

Comment:
I would recommend printing the `path_dir` to facilitate debugging :-)

Reply:
done in 
a3e884d